pkg/query/conv.go

```diff
@@ -1,10 +1,56 @@
 package query
 
 import (
+	"strconv"
+
 	sitter "github.com/smacker/go-tree-sitter"
 	"go.lsp.dev/protocol"
 )
 
+func Unquote(input []byte, n *sitter.Node) string {
+done:
+	for {
+		switch n.Type() {
+		case NodeTypeModule,
+			NodeTypeBlock,
+			NodeTypeExpressionStatement:
+			n = n.Child(0)
+		case NodeTypeString:
+			break done
+		default:
+			return ""
```

Comment:
Should this return an error instead? If you're not passing a thing containing a string literal I feel like this is the best we can do.